data/follpvosten.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Filename must match GH username exactly, including case

Reply:
Ooooh alright, I was wondering where the GitHub user name would come from. Thanks for the heads-up.